site/agenda.md

```diff
@@ -1,29 +1,38 @@
 Agenda
 ======
 
-## Saturday/Sunday (January 9th, 10th): Workshop kickoff
-
-Multiple, parallel sessions with the intent to bootstrap the workshop.
-These will be small group based (approximately 10 - 15 students) and short (about 15 minutes).
-Things to accomplish during this time:
-  * Say hello, tell them a little bit about Unidata, cover the general idea of the workshop
-  * Assign jupyterhub login information and make sure each student can get into the workshop environment.
-  * Cover the very basics of the jupyterhub environment:
-    * Exploring the interface and opening a notebook
-    * Executing cells
-    * Uploading an existing notebook / creating a new notebook
-  * Point them to a set of asynchronous workshop resources, which will include a list of project ideas for three skill levels: never programmed at all or no experience with python, basic knowledge of python, advanced python user. (these need to be better defined, for sure).
-
-Participants will be working in the Jypyterhub environment throughout the week, so these small group sessions will be critical.
-It could also be a good time to encourage students to pair up into groups to work on the project together.
-
-## Throughout the week of AMS (January 9th, 2021 — January 14th, 2021): Workshop
-
-This will generally be a period of asynchronous exploration for the participants.
-We will hold multiple formal "office hour" sessions throughout the week to assist participants directly.
-While Unidata staff will be monitoring email/social media for questions, having the office hours will provide some structure to the otherwise nebulous nature of this period of the workshop.
-Engaging with students using something like Google Hangout is what I'm thinking for the office hour help sessions.
-
-## Thursday (January 14th, 2021): Workshop showcase
-At the end of the week, Unidata will host a virtual workshop showcase.
-This might involve hosting a website where we can publish student notebooks, providing space for real-time discussions during a set time (multiple Google Hangouts, Zoom with breakout rooms), etc.
+## Sunday, January 10th, 2021 (A: 1:00 pm EST to 1:30 pm EST, B: 1:30 pm EST to 2:00 pm EST): Workshop kickoff
+
+We will offer two 30-minute sessions with the intent to kick-start the workshop.
+Each session will cover the same material, and each is limited to 100 participants.
+Interested students will need to register for this workshop using this Google Form [TBD].
+The only requirement is that you have a GitHub account prior to the workshop.
+No experience with GitHub or Python are necessary to participate!
+
+Each session will begin with a 10 minute overview, followed by a 15 minute, small group (10 students or less per group) breakout session.
+These sessions will take place in a Zoom meeting hosted by the American Meteorological Society.
+The following topics will be covered:
+  * asynchronous workshop resources
+  * accessing into the jupyterhub server
```

Comment:
"accessing into"? Maybe "access to"?

Reply:
I think I meant "accessing the". I can change it to "access to" if you would rather have it "access to". Either was is good by me!